scripts/perf/jenkins_script.sh

```diff
@@ -11,15 +11,46 @@ mkdir -p mnt
 
 #TODO: need this in the google cloud docker in order to auth
 #get startup script to run on new instance
-curl https://raw.githubusercontent.com/broadinstitute/cromwell/$CROMWELL_BRANCH/scripts/perf/startup_script.sh > mnt/startup_script.sh
+curl https://raw.githubusercontent.com/broadinstitute/cromwell/$CROMWELL_PERF_SCRIPTS_BRANCH/scripts/perf/startup_script.sh -o mnt/startup_script.sh
 
 DB_PASS=`docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN \
 	broadinstitute/dsde-toolbox vault read -format=json secret/dsp/cromwell/perf | jq '.data.db_pass'`
 
 docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN broadinstitute/dsde-toolbox vault read -format=json secret/dsp/cromwell/perf/service-account-deployer | jq -r '.data.service_account' > mnt/sa.json
+	
+function join { local IFS=","; echo "$*"; }
```

Comment:
`join() {` is supposedly more portable. http://tldp.org/LDP/abs/html/functions.html